src/logger.c

```diff
@@ -11,7 +11,7 @@
 LoggerData g_logger;
 
 #ifndef min
-#define	min(a,b)	(((a) < (b)) ? (a) : (b))
+#define min(a,b) (((a) < (b)) ? (a) : (b))
```

Comment:
Can't you just include the file that normally provides it within the header where you define it.